src/templates/shared/markdown.js

```diff
@@ -15,6 +15,10 @@ renderer.heading = function(text, level) {
 marked.setOptions({
   renderer,
   highlight: function(code, language) {
+    //clojure provides good highlighting for haloscript
+    if (language == "hsc") {
+      language = "clojure";
+    }
```

Comment:
Didn't feel this warranted a ticket of its own so slipping it in

Reply:
Using lisp here might get more accurate syntax highlighting. Clojure has some slightly wacky syntax and I worry that it could have some odd breakages because of it.

Reply:
I noticed that with lisp, boolean literals don't get highlighted specially whereas with clojure they do. I tested it with a50's script and it seemed OK.

Reply:
Well, that's odd. Clojure works for Halo script from the structure I've seen anyway